Tools/autotest/logger_metadata/parse.py

```diff
@@ -82,6 +82,7 @@ def set_vehicles(self, vehicles):
     def search_for_files(self, dirs_to_search):
         _next = []
         for _dir in dirs_to_search:
+            _dir = os.path.join(topdir, _dir)
```

Comment:
Have you tested parse.py when we're running the script which generates the logger documentation on the autotest server?

Reply:
the autotest server should use `Tools/scripts/build_log_message_documentation.sh` that is call from the root of the Ardupilot git directory.
top_dir is correctly set as the root of ardupilot git directory, so it is good ! 
This modification allow to always get back to the root directory and thus find the vehicles subdirectory !